tests/end_to_end/end_to_end.cpp

```diff
@@ -1074,6 +1074,82 @@ map_test(ebpf_execution_type_t execution_type)
     bpf_object__close(unique_object.release());
 }
 
+void
+global_var_test(ebpf_execution_type_t execution_type)
```

Comment:
```suggestion
global_variable_test(ebpf_execution_type_t execution_type)
```
avoid unnecessary abbreviations